app-modules/integration-monday/docs/integration-guide.html

```
@@ -0,0 +1,617 @@
+
+<section>
+  <div class="wrap">
+    <p class="eyebrow reveal">Por que não entra em loop</p>
+    <h2 class="reveal">Idempotência, sem flag global</h2>
+    <p class="section-sub reveal">
+      A mudança vinda da Monday também dispara o push de volta — e mesmo assim não há loop.
+    </p>
+    <div class="duo">
+      <div class="mini reveal">
+        <h4>O eco do status morre sozinho</h4>
+        <p>O push reescreve o <span class="k">mesmo</span> status. Se a Monday devolver um webhook, o ticket já está naquele estado → <span class="k">$ticket->status === $status</span> → ignora.</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)